doc/enterprise/auth/saml/saml_setup.md

```diff
@@ -0,0 +1,244 @@
+# Configure a SAML application
+
+Pachyderm supports Security Assertion Markup Language
+(SAML) providers to authenticate with Pachyderm which enables
+you to configure Single Sign-On (SSO) by using browser tokens.
```

Comment:
I would phrase this differently: "Pachyderm allows users to authenticate with Pachyderm via SAML ID providers (such as Okta)".

(the ID provider is just the mechanism through which the authentication is happening. The entity that's authenticating with pachyderm is just a person. The difference is meaningful because users do have to configure Pachyderm to trust the ID provider, and in that step, the ID provider itself *is* authenticating with Pachyderm, but that's not the end goal)